fed/api.py

```diff
@@ -131,6 +133,14 @@ def init(
         cross_silo_serializing_allowed_list: The package or class list allowed for
             serializing(deserializating) cross silos. It's used for avoiding pickle
             deserializing execution attack when crossing solis.
+        cross_silo_send_resource_label: Customized resource label, the SendProxyActor
```

Comment:
If we are trying to limit the user facing proxy configs, maybe exporse ProxyActorConfig would give us more extensibility？If we would like to add another config, this might look tedious, also, send_max_retries could merge into ProxyActorConfig?

Reply:
+1.
And I vote for a name `ProxyConfig`, which not exposing more details on the ray actor implementation.

Reply:
Thx for the advice~  @fengsp @jovany-wang 

That's also what I plan to do in [2/N]:  merge cross-silo RPC related config into ProxyActorConfig, and then expose it as a parameter of `fed.init`.

[1/N] is just trying to enable placing the ProxyActor according to its resource label.

@jovany-wang `ProxyConfig` is indeed a better name when exposing it to users 👍